src/Servers/IIS/IIS/src/IISServerOptions.cs

```diff
@@ -8,6 +8,9 @@
 
 namespace Microsoft.AspNetCore.Builder
 {
+    /// <summary>
+    /// Options to configure IIS In-Process with.
```

Comment:
Do we have examples of good options docs? This feels like the min bar.

Reply:
I looked through ~10 options types and pretty much all top level doc comments are like this.